src/rshim.c

```diff
@@ -2069,43 +2120,40 @@ rshim_backend_t *rshim_find_by_dev(void *dev)
   return NULL;
 }
 
+/*
+ * Must run with bd->mutex held.
+ *
+ * No need to call this function when value is set to 1. Just set bd->drop_mode
+ * to 1.
```

Comment:
Can we remove these three lines. since we could still call this function even value is 1.